src/main/java/seedu/address/ui/StatusBarFooter.java

```diff
@@ -17,6 +20,7 @@
 
     public static final String SYNC_STATUS_INITIAL = "Not updated yet in this session";
     public static final String SYNC_STATUS_UPDATED = "Last Updated: %s";
+    public static final String TARGET_COUNT_STATUS = "%d targets";
```

Comment:
Maybe a better string would be "Found %d persons"?

Reply:
Nice suggestion indeed!